anticipy/forecast_plot.py

```diff
@@ -357,6 +357,10 @@ def plot_forecast(df_fcast, path, output='html', width=None,
 
     assert isinstance(df_fcast, pd.DataFrame)
 
+    if not path and (output == 'html' or output == 'png'):
+        logger.error('No export path provided.')
+        return 0
```

Comment:
https://stackoverflow.com/questions/9426045/difference-between-exit0-and-exit1-in-python

Reply:
thanks and done!